perf-event/src/events.rs

```diff
@@ -257,6 +264,19 @@ impl Cache {
     }
 }
 
+/// A Raw event
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct Raw {
+    /// Raw config of the event
+    pub config: u64,
```

Comment:
Shouldn't this also include `config1` and `config2`? I'm not sure where `config2` is used but `config1` is used for some offcore x86 events.

Reply:
Added support for setting config1 and config2.